consensus/src/liveness/multi_proposer_test.rs

```diff
@@ -49,17 +49,27 @@ fn test_multi_proposer() {
     }
     assert_eq!(pe.is_valid_proposer(proposers[another_idx], round), None);
 
-    let good_proposal =
-        Block::new_proposal(1, 1, 1, certificate_for_genesis(), &signers[primary_idx]);
+    let good_proposal = Block::new_proposal(
```

Comment:
should we make an empty_proposal?

Reply:
it just saves one field, we still need to fill in other fields